src/webln/NostrWeblnProvider.ts

```diff
@@ -213,6 +236,40 @@ export class NostrWebLNProvider {
       });
     });
   }
+
+  initNWC(provider: string, options: { name: string, returnTo?: string }) {
+    const height = 600;
+    const width = 400;
+    const top = window.outerHeight / 2 + window.screenY - height / 2;
+    const left = window.outerWidth / 2 + window.screenX - width / 2;
+    // @ts-ignore for my TS love :)
+    const url = new URL(NWC_URLS[provider]);
+    url.searchParams.set('c', options.name);
```

Comment:
`c` is for client right? what if the client wants to generate multiple connection strings? would they just add a suffix or something?

Reply:
why would a client want multiple connections? 
the c is just the app name passed to NWC.

Reply:
@bumi yeah exactly so you know what session is being used for so that you e.g. don't accidentally delete the wrong session. Another example might be lightsats if we had multiple tips. It might be good to be able to differentiate them somehow, maybe a description field or something?